.github/stale.yml

```diff
@@ -0,0 +1,18 @@
+---
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
+# Label to use when marking an issue as stale
+staleLabel: wontfix
```

Comment:
Instead of `wontfix` shouldn't we set the label as `stale` itself?

Reply:
yeah, good catch.  I copied this from another repo, will make the change.